ci/bamboo/build-osx.sh

```diff
@@ -1,32 +1,40 @@
 #!/bin/bash
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+#
+# OPTIONS:
+#  --user: If provided, will pass along the --user argument to any pip install
```

Comment:
I don't prefer it, I was trying to keep the same interface for this set of scripts. That interface made it seem like command line arguments were sent in from outside, because they are all passed from the OS X script to the generic script with the `$@` symbol. 

I am happy to change this, and I will right now.